go-controller/pkg/ovn/multicast_test.go

```diff
@@ -169,28 +169,28 @@ func getMulticastDefaultStaleData(clusterPortGroup, clusterRtrPortGroup *nbdb.Po
 
 func getDefaultPortGroups() (clusterPortGroup, clusterRtrPortGroup *nbdb.PortGroup) {
 	clusterPortGroup = &nbdb.PortGroup{
-		UUID: types.ClusterPortGroupName + "-UUID",
-		Name: types.ClusterPortGroupName,
+		UUID: types.ClusterPortGroupNameBase + "-UUID",
+		Name: types.ClusterPortGroupNameBase,
 		ExternalIDs: map[string]string{
-			"name": types.ClusterPortGroupName,
+			"name": types.ClusterPortGroupNameBase,
 		},
 	}
 	clusterRtrPortGroup = &nbdb.PortGroup{
-		UUID: types.ClusterRtrPortGroupName + "-UUID",
-		Name: types.ClusterRtrPortGroupName,
+		UUID: types.ClusterRtrPortGroupNameBase + "-UUID",
+		Name: types.ClusterRtrPortGroupNameBase,
 		ExternalIDs: map[string]string{
-			"name": types.ClusterRtrPortGroupName,
+			"name": types.ClusterRtrPortGroupNameBase,
 		},
 	}
 	return
 }
 
-func getMulticastPolicyExpectedData(ns string, ports []string) []libovsdb.TestData {
-	pg_hash := getMulticastPortGroupName(ns)
-	egressMatch := getACLMatch(pg_hash, getMulticastACLEgrMatch(), aclEgress)
+func getMulticastPolicyExpectedData(fakeController *DefaultNetworkController, ns string, ports []string) []libovsdb.TestData {
+	pg_hash := fakeController.getMulticastPortGroupName(ns)
```

Comment:
we usually use
`fakeController = getFakeController(DefaultNetworkControllerName)` right here in the function to avoid adding arguments (of course in case the controller type is predefined)